great_expectations/expectations/core/expect_column_median_to_be_between.py

```diff
@@ -298,6 +298,8 @@ def _prescriptive_renderer(  # type: ignore[override] # TODO: Fix this type igno
                 template_str = f"median must be {at_most_str} $max_value."
             elif params["max_value"] is None:
                 template_str = f"median must be {at_least_str} $min_value."
+            else:
+                raise ValueError("unresolvable template_str")
```

Comment:
This is the biggest runtime change I've made.
We now raise this `ValueError` if this `template_str` is not defined.
We would have hit an error anyways, but this does it earlier.